transcendence-app/src/chat/chatroom/chatroom-member/infrastructure/db/postgres/chatroom-member.postgres.repository.ts

```diff
@@ -32,14 +34,13 @@ export class ChatroomMemberPostgresRepository
     const joinedAtColName = ChatroomMemberKeys.JOINED_AT;
 
     const chatroomMembersData = await makeQuery<ChatroomMember>(this.pool, {
-      text: `INSERT INTO ${this.table} (${cols.join(
-        ',',
-      )}) values (${params.join(',')})
-      ON CONFLICT
-      (${ChatroomMemberKeys.CHATID}, ${ChatroomMemberKeys.USERID})
-      DO UPDATE
-      SET ${joinedAtColName} = EXCLUDED.${joinedAtColName}
-      RETURNING *;`,
+      text: `INSERT INTO ${this.table} (${cols.join(',')})
```

Comment:
praise: Thank you! Do you use any plugins for auto-formatting?

Reply:
Sort of